wandb/errors/__init__.py

```diff
@@ -60,7 +101,9 @@ class LaunchError(Error):
     "UsageError",
     "CommError",
     "LogError",
+    "DockerError",
     "LogMultiprocessError",
+    "ShellCommandError",
```

Comment:
This is unused currently, is it intended to replace any existing errors?

Reply:
Nope I think it was being used at some point, we could get rid of it.